jetty-core/jetty-staticapp/src/main/java/org/eclipse/jetty/staticapp/StaticAppContext.java

```diff
@@ -40,35 +40,35 @@
  *     This is basically just a {@link ResourceHandler} with a context-path.
  * </p>
  */
-public class StaticContextHandler extends ContextHandler
+public class StaticAppContext extends ContextHandler
```

Comment:
The javadoc here could also be improved: A specialised ContextHandler that provides a Web Application that serves only static content using Jetty core components